html/pfappserver/root/static.alt/src/views/Configuration/_config/authenticationSource.js

```diff
@@ -685,6 +685,19 @@ export const viewFields = {
       ]
     }
   },
+  dead_duration: (form, meta = {}) => {
+    return {
+      label: i18n.t('Dead duration'),
+      text: i18n.t('How much time in seconds should a server be marked dead before it is retried. When specifying multiple LDAP servers or a DNS name pointing to multiple IPs, then this option can be used to offer more consistent failover. A value of 0 disables this feature.'),
```

Comment:
Same

Reply:
The translation will happen pre-release like it always does

Reply:
If it will be backported. It should be translated.

Reply:
Lets get someone comfortable with the translation on this then if its important to the merge of this PR since I'm not familiar with that